site/pages/sysadmin/users.page.tsx

```diff
@@ -38,20 +39,23 @@ const AdminUsers = (props: AdminUserProps): ReactElement => {
 
     const getRows = () => {
         const rows: { header?: string | ReactNode; cells: string[] | ReactNode[] }[] = [];
-        props.admins?.forEach((user, index) => {
+        props.users?.forEach((user, index) => {
```

Comment:
Could the data be sorted according to first name or email? Chris or Anna should be able to confirm

Reply:
refactored and created sortedAndFilteredUsersList to sort users by first name when we perform the filter action